ob-pattern-1/README.md

```diff
@@ -0,0 +1,95 @@
+# Helm Chart for deployment of WSO2 Open Banking API Manager with WSO2 Open Banking Key Manager
+
+![WSO2 Open Banking Deployment Pattern 1](images/pattern1.png)
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+
+## Prerequisites
+
+* In order to use WSO2 Open Banking Helm resources, you need an active WSO2 Open Banking subscription. If you do not possess an active WSO2 Open Banking subscription already, you can contact us from [here](https://wso2.com/solutions/financial/open-banking/).<br><br>
+ 
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (compatible with v1.10) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/).<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). This can be easily done via
+ ################
+  ```
+  helm install stable/nginx-ingress --name nginx-wso2ob --set rbac.create=true
+  ```
+
+## Quick Start Guide    
+
+>In the context of this document, <br>
+>* `HELM_HOME` will refer to a local copy of the [`wso2/kubernetes-open-banking`](https://github.com/wso2/kubernetes-open-banking/)
+Git repository. <br>
+
+##### 1. Clone Kubernetes Resources for WSO2 Open Banking Git repository.
+
+```
+git clone https://github.com/wso2/kubernetes-open-banking.git
+```
+
+##### 2. Setup persistent storage.
```

Comment:
@kdanW we don't need multiple steps to install the dependencies such as, the MySQL Helm chart and NFS server provisioner (please see [1] for an example).

For this purpose, please use the `requirements.yaml`.

[1]: [Release `v5.9.0.1` of IAM Helm resources](https://github.com/wso2/kubernetes-is/tree/v5.9.0.1/advanced/is-pattern-1#3-deploy-wso2-identity-server)

Reply:
We will have to host MySQL chart for OB in WSO2 Helm repo for that. Shall we keep this as it is until we do that?

Reply:
@kdanW when you create a `<HELM_HOME>/requirements.yaml`, initially it checks for the presence of the referred Charts at the `<HELM_HOME>/charts` directory. So you can manually create a `<HELM_HOME>/charts` directory, package/download the dependencies and place them in it.

These steps are not required.

Reply:
@kdanW this step is not required.

Yes, this is required once you need to instruct on how to setup external NFS (e.g. [1]).

But it is not required, in the case of using the NFS Server Provisioner.

[1]: [Setting Up Persistent Storage](https://github.com/wso2/kubernetes-apim/tree/v3.0.0.1/advanced/am-pattern-1#2-setup-persistent-storage)

Reply:
Fixed in https://github.com/wso2/kubernetes-open-banking/pull/22/commits/f548e4dfe3b0ad73fa05d8858b312ca0494f1589